synfig-studio/src/gui/app.cpp

```diff
@@ -1057,13 +1057,28 @@ DEFINE_ACTION("help-about",     Gtk::StockID("synfig-about"))
 // actions: Keyframe
 DEFINE_ACTION("keyframe-properties", _("Properties"))
 
+// actions: move to tab
+for (int i = 1; i <= 8; ++i) {
+	const std::string tab = std::to_string(i);
+	DEFINE_ACTION("move-to-tab-" + tab, _("Move to Tab ") + tab)
```

Comment:
1. I think may be better to replace the word "Move" with "Switch" or "Jump" as a more established term?

https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Ctab-window-shortcuts

https://workona.com/blog/switch-tabs-chrome-shortcut/#:~:text=The%20keyboard%20shortcut%20Cmd%20%2B%20Shift,switch%20between%20tabs%20in%20Chrome.

2. I noticed the term "Jump to the rightmost tab" in Google help. For right-to-left languages, will the last tab be on the left or right? Also "last" is not the best word here, because the user may be thinking about the last tab he worked on.

Reply:
1- sure, looks like switch is more established and appropriate will be doing this.

2- I think regarding right to left languages (as is Arabic which I  speak) rightmost shouldnt realy be a problem the las tab is still the rightmost tab. I even checked ( to maybe see if in synfig order is reversed upon changing to this type of language -which would be not standard behaviour-) but even in arabic still the rightmost tab is the last tab, so I think its ok. 
p.s. yes last could posssibly cause confusions so rightmost would probably be bettter ill also check how they named it in other browsers.

Reply:
done.